logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/UserSessionScope.kt

```diff
@@ -474,6 +473,10 @@ class UserSessionScope internal constructor(
     private val logoutCallback: LogoutCallback,
 ) : CoroutineScope {
 
+    init {
+        onUserSessionScopeCreated()
```

Comment:
Small refactor to have better understanding (and kdocs) of what work we do subscribe when we create a session in the app.